### DISCUSSION TITLE
vue-web-terminal/questions

### DISCUSSION THREAD

**Discussion by giscus** [tzfun/vue-web-terminal#121] [Announcements]
# vue-web-terminal/questions

A powerful web-side command line window plugin

https://tzfun.github.io/vue-web-terminal/questions.html

**Comment by river666** [comment]
Can I change this path "/vue-web-terminal/ "?
I can't found API to change this path.

**Comment by tzfun** [comment]
@river666 You need to bind a [context](https://tzfun.github.io/vue-web-terminal/attributes.html#context) attribute and then modify its value.

**Reply by river666** [reply]
thks!
I hope display like this picter
![image](https://github.com/user-attachments/assets/86466da5-1bb2-42db-b3cc-d4b21b3e958f)
Actual is:
![image](https://github.com/user-attachments/assets/0d8141c6-796d-444f-8b7a-3c8bac9bafab)
This was my response:
![image](https://github.com/user-attachments/assets/febafb00-4452-412c-94f8-266a4a53489f)

**Reply by river666** [reply]
What should I do

**Reply by tzfun** [reply]
> What should I do

I think you need to read this [description](https://tzfun.github.io/vue-web-terminal/about.html#what-it-can-t-do) first. 

Your content contains some ANSI codes for formatting. This plugin only supports ANSI codes for color rendering, so it is not feasible for you to output it directly.

If you don't mind the trouble, you can also format the output content yourself in other ways, such as splitting this content and outputting it in table format.

**Reply by river666** [reply]
![image](https://github.com/user-attachments/assets/9f38a6ea-2d27-4ffb-b3c5-6dd18d8b3534)
tks!

**Comment by Art3misXR** [comment]
How do you have codemirror working with code and edit features like in the online experience? Is there somewhere I can download the online experience source?

**Reply by tzfun** [reply]
> How do you have codemirror working with code and edit features like in the online experience? Is there somewhere I can download the online experience source?

You need to use [slot](https://tzfun.github.io/vue-web-terminal/slots.html) to implement a third-party editing plugin like Codemirror. The relevant code in the Demo is in [branch doc LocalTerminal.vue](https://github.com/tzfun/vue-web-terminal/blob/doc/docs/.vuepress/components/local-terminal/LocalTerminal.vue)
